docs/04-documentation.md

```diff
@@ -0,0 +1,46 @@
+---
+title: How to update documentation on ECL
+label: Edit documentation
+---
+
+This page explains how to update documentation directly on GitHub.
+
+## Prerequisite
+
+If you don't already have a GitHub account, make sure to create one and log in with it.  
+You also have to be part of team [ECL doc editors](https://github.com/orgs/ec-europa/teams/ecl-doc-editors/).
+
+## Step 1: Open documentation file
+
+- Go to [ECL repository](https://github.com/ec-europa/europa-component-library)
+- Locate the component you wish to edit. Components are located in `framework/components`
```

Comment:
What if the person wants to edit the documentation for a utility? or a template?

Reply:
For now they should update components only I guess (for DT)

Reply:
Let's start with components, we'll see afterwards what is actually needed